### ISSUE TITLE
Add a `ceph.rbd.du` storage option to disable the use of `rbd du`

### ISSUE THREAD

**Issue by stgraber** [lxc/lxd#9348]
Running `rbd du` can be very resource intensive so there are users who would prefer to miss the disk usage data for stopped instances instead of dealing with the performance hit.

This should be a pretty easy change with just a small addition to the ceph driver for the config key and associated API extension and doc entry.

**Comment by liamnickell** [comment]
Hi, I'm a CS major at UT Austin and am interested in working on this feature. I also commented on [this issue](https://github.com/lxc/lxd/issues/9489#issuecomment-964689991) for reference.

**Comment by stgraber** [comment]
Assigned it to you!

**Comment by liamnickell** [comment]
@stgraber Hey, I think I'm about done on this issue but just wanted to clarify something. 

Should the only change to the ceph driver be that if `ceph.rbd.du` is set to disabled/false, then calls to `GetVolumeUsage` will always result in using the filesystem stats to obtain usage info (using `unix.Statfs`)?

**Comment by stgraber** [comment]
Nope, that should be it. For unmounted volumes, I believe we could return an error without it causing an actual error to the user, so that's probably best.
